consent/consent_record.go

```diff
@@ -162,7 +120,6 @@ func (r *RecordCreate) Parse(parser structure.ObjectParser) {
 
 func (r *RecordCreate) Validate(validator structure.Validator) {
 	validator.String("ageGroup", structure.ValueAsString(&r.AgeGroup)).OneOf(structure.ValuesAsStringArray(AgeGroups())...)
-	validator.Time("createdTime", &r.CreatedTime).NotZero().BeforeNow(time.Second)
```

Comment:
This doesn't come from the user, so doesn't need to be validated here.

I've moved responsibility for this to the repository, which checks when creating if it exists, and if not, sets it to time.Now().

Reply:
This validator makes sure the model that's inserted in the database is valid right before it's persisted.